Formula/apache-zeppelin.rb

```diff
@@ -0,0 +1,19 @@
+class ApacheZeppelin < Formula
+  desc "Multi-purpose Notebook supporting Spark, Cassandra etc..."
```

Comment:
We forbid descriptions that start with an indefinite article so `Web-based notebook that enables interactive data analytics`.